.github/actions/derive-artefact-address/action.yaml

```diff
@@ -0,0 +1,31 @@
+name: "Collect artefact address action"
+description: "Collects the artefact sub directory and commit has for pipeline being executed"
+
+inputs:
+  tag:
+      description: "The git tag to derive artefact location from"
+      required: true
+outputs:
+  commit_hash:
+    description: "Commit hash for head of branch being processed"
```

Comment:
Feel free to ignore but i think this is the commit hash relating to the git tag - it is not necessarily the hash at the head of the branch

Reply:
have updated description to suit the use better